src/System.Net.WebHeaderCollection/src/Resources/Strings.resx

```diff
@@ -138,13 +135,4 @@
   <data name="net_WebHeaderInvalidNonAsciiChars" xml:space="preserve">
     <value>Specified value has invalid non-ASCII characters.</value>
   </data>
-  <data name="net_WebHeaderMissingColon" xml:space="preserve">
-    <value>Specified value does not have a ':' separator.</value>
-  </data>
-  <data name="net_headerrestrict" xml:space="preserve">
-    <value>The '{0}' header must be modified using the appropriate property or method.</value>
-  </data>
-  <data name="net_emptystringcall" xml:space="preserve">
-    <value>The parameter '{0}' cannot be an empty string.</value>
-  </data>
 </root>
```

Comment:
There is a newline missing at the end of this file.

Also, why are these error message strings being removed?